src/developer/web-api/tracker.md

```diff
@@ -217,38 +217,36 @@ Tracker notes do not have their dedicated endpoint; they are exchanged as part o
 ```json
 {
   "trackedEntity": "oi3PMIGYJH8",
-  <entity_details>,
-  ],
+  "_comment" : "tracked entity details ...",
```

Comment:
Why do you add `_comment`?

Reply:
It Is the only way to add comment in JSON I see and it is better than the current solution. It is just an idea if we can come up with something different it is ok for me

Reply:
if "comment" is free to use, why do we add the underscore?

Reply:
why do we even need this comment? `"tracked entity details ..."` isn't all that useful. Adding the field can mislead someone to believe its part of the payload. I would remove it.

Also this will be removed once we publish the OpenAPI spec to which we have added descriptions.